lib/src/converters/converter.dart

```diff
@@ -690,6 +690,68 @@ const Converter<Mentionable> mentionableConverter = FallbackConverter(
   type: CommandOptionType.mentionable,
 );
 
+IAttachment? snowflakeToAttachment(Snowflake id, IChatContext context) {
+  Iterable<IAttachment>? attachments;
```

Comment:
I see there's a pattern everywhere in project why typed variables instead of final? Final is recomended in guidelines

Reply:
[This style recommendation](https://dart.dev/guides/language/effective-dart/usage#do-follow-a-consistent-rule-for-var-and-final-on-local-variables) recommends using `final` and/or `var`. I think that it's better to type variables, for two main reasons:

1. Make it clear what the variable is, as well as preventing type inference from getting the wrong type;
2. Readability. In the event that type inference doesn't work, using final would result in having to type `final <type> <name>` which is considerably longer than just `<type> <name>`.